tidy3d/components/data/monitor_data.py

```diff
@@ -434,6 +482,17 @@ def dot(self, field_data: ElectromagneticFieldData, conjugate: bool = True) -> x
         """Inner product is not defined for time-domain data."""
         raise DataError("Inner product is not defined for time-domain data.")
 
+    @property
+    def time_reversed_copy(self) -> FieldTimeData:
```

Comment:
just a comment that this looks almost identical to the method for FieldData, except without the conj. I wonder if we should just add this method to the ElectromagenticFieldDataset more generally. Note: `np.conj(float) -> float` so can just use the same logic as FieldData here.

Reply:
Time data can be complex if Bloch boundaries are present (although we had this discussion of whether it's actually needed... but for now we return it as complex). Also with the reversal of the `t` coordinate, I guess it makes sense to keep overwriting here?

Reply:
sorry, I dont quite follow. So for complex-valued time-domain data are you saying that we dont want to apply complex conjugate for time reversal? My point was that probably we should apply np.conj to all data (including real and complex) + reverse the time coordinates and that we can likely do this with one method at the ElectromagneticFieldData level.

Reply:
Yeah I don't think complex time-domain data has to be conjugated. Time reversal is just something like `t -> -t`. In the frequency domain, however, that flips the sign in `exp(-1j w t)` in the Fourier transform, hence the complex conjugation.

Reply:
ok, I would have assumed you would need to CC even in the complex-valued time domain formalism but I trust you if you say you dont. I figured the same logic applies that you could get time domain fields through FR eg `E(t) = ∫dw E(w) exp(-iwt)` and therefore would need to CC E(t) if t -> T-t